lib/spack/docs/module_file_support.rst

```diff
@@ -367,14 +367,14 @@ and has similar effects on module file of dependees. Even in this case
 
   with the following snippet:
 
-  .. literalinclude:: ../../../var/spack/repos/builtin/packages/R/package.py
+  .. literalinclude:: ../../../var/spack/repos/builtin/packages/r/package.py
      :pyobject: R.setup_environment
 
   The ``R`` package also knows which environment variable should be modified
```

Comment:
@citibeth This should be `r`. You might want to look through the documentation to make sure there aren't other capitalized packages being reference. The documentation tests aren't foolproof.